### ISSUE TITLE
Add-on error for OpenShift version semantics can be improved

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#1603]
Currently the error is `Add-on only support OpenShift version semantics eg. 3.6.0 or >3.6.0, <3.9.0 or >=3.5 etc.` and I think different versions in the message is confusing (at least to me) and we should fix it. May be we can even try to come up with a better message. @minishift/minishift-dev WDYT?

```
if !checkVersionSemantic(headers[RequiredOpenShiftVersion].(string)) {
      return nil, errors.New("Add-on only support OpenShift version semantics eg. 3.6.0 or >3.6.0, <3.9.0 or >=3.5 etc.")
}
```

**Comment by budhrg** [comment]
Agreed :+1:

**Comment by tylerauerbeck** [comment]
This one looks a little old, so I'm not sure if there was a decision made elsewhere on the wording of this updated message. If not, would it make sense to use something like

`Add-on syntax uses Openshift version semantics. (i.e. assuming the use of 3.6.0, it would accept 3.6.0, >3.6.0, <3.6.0, >=3.6 etc.)`

I agree with the above that using different versions makes this a little more confusing than necessary, so explicitly calling out that an example uses version X and then gives examples using version X would make this a bit more clear.

**Comment by LalatenduMohanty** [comment]
Resolved via #2038
